crates/moss-store/src/experiments/adaptive.rs

```diff
@@ -0,0 +1,160 @@
+use cached::{Cached, CanExpire, ExpiringValueCache};
+use std::any::Any;
+use std::collections::HashMap;
+use std::fmt::Debug;
+use std::time::{Duration, Instant};
+use tracing_mutex::stdsync::Mutex;
```

Comment:
In the project, I always try to use `parking_lot` for Mutex/RwLock. If there’s no specific need to use something else, it’s better to stick with `parking_lot`.